pub-matic/pub-matic-htb.js

```diff
@@ -335,6 +336,10 @@ function PubMaticHtb(configs) {
                     gdpr: gdprStatus.applies ? 1 : 0
                 }
             };
+            if (uspConsentObj) {
+                // eslint-disable-next-line camelcase
+                payload.regs.ext.us_privacy = uspConsentObj.uspString;
```

Comment:
FYI - Please note the `uspString` property will be set as an empty string in the scenarios where the IX Library is not able to retrieve consent/opt-out from the site's page.